backend/de.metas.acct.base/src/main/java/de/metas/acct/gljournal_sap/service/SAPGLJournalCreateRequest.java

```diff
@@ -59,28 +60,30 @@
 
 	@NonNull ImmutableList<SAPGLJournalLineCreateRequest> lines;
 
+	@Nullable SAPGLJournalId reversalId;
+
 	@NonNull
 	public static SAPGLJournalCreateRequest of(
 			@NonNull final SAPGLJournal journal,
-			@NonNull final Instant dateDoc,
-			final boolean negateAmounts)
```

Comment:
we still need the `negate amounts` or `reverse accounts` boolean, as building a `SAPGLJournalCreateRequest` from an existing `SAPGLJournal` doesn't automatically imply it's a reversal.

Reply:
correction done